cloud-controller-manager/cmd/digitalocean-cloud-controller-manager/main.go

```diff
@@ -17,51 +17,40 @@ limitations under the License.
 package main
 
 import (
+	"flag"
 	"fmt"
 	"os"
 
 	"k8s.io/apiserver/pkg/server/healthz"
-	"k8s.io/apiserver/pkg/util/flag"
-	"k8s.io/apiserver/pkg/util/logs"
+	"k8s.io/component-base/logs"
 	"k8s.io/kubernetes/cmd/cloud-controller-manager/app"
-	"k8s.io/kubernetes/cmd/cloud-controller-manager/app/options"
 	_ "k8s.io/kubernetes/pkg/client/metrics/prometheus" // for client metric registration
 	_ "k8s.io/kubernetes/pkg/version/prometheus"        // for version metric registration
-	"k8s.io/kubernetes/pkg/version/verflag"
 
 	_ "github.com/digitalocean/digitalocean-cloud-controller-manager/cloud-controller-manager/do"
-	"github.com/golang/glog"
-	"github.com/spf13/pflag"
 )
 
 func init() {
 	healthz.DefaultHealthz()
 }
 
 func main() {
-	s, err := options.NewCloudControllerManagerOptions()
-	if err != nil {
-		glog.Fatalf("failed to create config options: %s", err)
-	}
+	// Bogus parameter needed until https://github.com/kubernetes/kubernetes/issues/76205
+	// gets resolved.
+	flag.CommandLine.String("cloud-provider-gce-lb-src-cidrs", "", "NOT USED (workaround for https://github.com/kubernetes/kubernetes/issues/76205)")
 
-	s.AddFlags(pflag.CommandLine)
+	command := app.NewCloudControllerManagerCommand()
 
-	flag.InitFlags()
+	// (The following comment is copied from upstream:)
+	// TODO: once we switch everything over to Cobra commands, we can go back to calling
+	// utilflag.InitFlags() (by removing its pflag.Parse() call). For now, we have to set the
+	// normalize func and add the go flag set by hand.
+	// utilflag.InitFlags()
 	logs.InitLogs()
 	defer logs.FlushLogs()
 
-	verflag.PrintAndExitIfRequested()
-
-	// digitalocean overrides
-	s.KubeCloudShared.AllowUntaggedCloud = true
```

Comment:
@timoreimann and I had an offline discussion about this.  Because of the upstream changes were no longer able to do this, but it sounds like we need to change the parameters required to invoke the CCM.  We should also update the documentation in this PR so they ride together.

Reply:
Good catch -- I pushed a commit that updates the docs. PTAL again.